test/performance/README.md

```diff
@@ -0,0 +1,25 @@
+# Profiles Performance Test
+
+The `profiles.jmx` [JMeter](http://jmeter.apache.org/) test will call the Profiles application for a number of different surgeries.
+Several parameters are exposed to customise the base test.
+
+
+| Parameter                         | Description                                                        | Default               |
+|:---------------------------------|:-------------------------------------------------------------------|-----------------------|
+| `hostname`                       | URL of server to test                                              | profiles.dev.beta.nhschoices.net|
+| `users`                          | Simulated number of concurrent users                               | 5                  |
+| `rampup`                         | Time in seconds to ramp up to total number of users                | 20                 |
+| `duration`                       | Time in seconds to run test                                        | 120                |
+| `throughput`                     | target throughput in samples per minute                            | 120                |
+| `csvfile`                        | csv data file of choices ID to use, either `ids_100.csv`, `ids_1200.csv`, or `ids_all.csv` (9000+), allowing a number of different GP pages to be visited during the test | ids_100.cs         |
+
+## Running the Test
+
+The test may be loaded into the JMeter application, or the file may be run from the command line locally by changing into the `test/performance` folder and entering:
+
+/usr/local/bin/jmeter/jmeter -n -t  ./profiles.jmx -Jhostname=*hostname* -Jusers=*users* -Jrampup=*rampup* -Jduration=*duration* -Jthroughput=*throughput* Jcsvfile=*csvfile* -l profiles.jtl
+
+
+Where the parameters in italics can be substituted with the desired value, or removed entirely to use the default.
+
+*Note:* The path to JMeter may differ depending on your installation location.
```

Comment:
The goal of this PR is to add a JMeter tests to be used by the Teamcity performance dashboards. The information in the readme is to describe the parameters used, and provide assistance if you want to run them locally. 

It's not meant to replace any other approaches, just to describe how you might run them if you'd like to.